File: app/src/main/java/com/lifedawn/bestweather/data/remote/retrofit/responses/metnorway/locationforecast/timeseries/Summary.java
package com.lifedawn.bestweather.data.remote.retrofit.responses.metnorway.locationforecast.timeseries;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Summary {
	@Expose
	@SerializedName("symbol_code")
	private String symbolCode;
	
	public String getSymbolCode() {
		return symbolCode;
	}
	
	public void setSymbolCode(String symbolCode) {
		this.symbolCode = symbolCode;
	}
}

File: app/src/main/java/com/lifedawn/bestweather/data/remote/retrofit/responses/metnorway/locationforecast/timeseries/Summary.kt
package com.lifedawn.bestweather.data.remote.retrofit.responses.metnorway.locationforecast.timeseries

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Summary {
    @Expose @SerializedName("symbol_code") var symbolCode: String? = null
}